libsolidity/interface/Natspec.cpp

```diff
@@ -100,15 +100,52 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 		{
 			Json::Value method(devDocumentation(fun->annotation().docTags));
 			if (!method.empty())
+			{
 				// add the function, only if we have any documentation to add
+				Json::Value ret(Json::objectValue);
```

Comment:
Could use uniform initialization and maybe a more descriptive name e.g. `Json::Value jsonReturn{Json::objectValue}`